app/javascript/mastodon/locales/zh-TW.json

```diff
@@ -29,6 +29,9 @@
   "column_back_button.label": "上一頁",
   "column_subheading.navigation": "瀏覽",
   "column_subheading.settings": "設定",
+  "community.column_settings.advanced": "進階",
```

Comment:
Since all of these are the same, could you just rename the original key to something more generic than `home.column_settings.advanced` and then use that everywhere?